k8s/aat/common/ia/home-office-mock-api.yaml

```diff
@@ -0,0 +1,29 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ia-home-office-mock-api
+  namespace: ia
+  annotations:
+    fluxcd.io/automated: "true"
+    repository.fluxcd.io/java: java.image
+    filter.fluxcd.io/java: glob:prod-*
+spec:
+  releaseName: ia-home-office-mock-api
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: ia-home-office-mock-api
+    version: 0.0.1
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/ia/home-office-mock-api:latest
+      ingressHost: ia-home-office-mock-api-aat.service.core-compute-aat.internal
```

Comment:
you have `ingressHost` in values.yaml file in application. I think it is not needed here

Reply:
Changed